go/launcher/webdriver/webdriver.go

```diff
@@ -404,7 +416,7 @@ func processResponse(body io.Reader, value interface{}) (*jsonResp, error) {
 		return nil, errors.New(compName, fmt.Errorf("%v unmarshalling %q", err, respBody))
 	}
 
-	if errBody.Status == 0 && errBody.Error == "" {
+	if respBody.Status == nil || *respBody.Status == 0 {
```

Comment:
I don't understand what is happening in the function :-)

I rewrote it to make the logic easier to follow and reason about. Additionally, I moved the code for checking if a jsonResp is an error into a method: isError.